substrate/bin/node/runtime/src/lib.rs

```diff
@@ -2747,6 +2747,12 @@ impl_runtime_apis! {
 			// have a backtrace here.
 			Executive::try_execute_block(block, state_root_check, signature_check, select).unwrap()
 		}
+
+		fn decode_entire_state() {
+			// NOTE: intentional unwrap: we don't want to propagate the error backwards, and want to
+			// have a backtrace here.
+			Executive::try_decode_entire_state().unwrap()
```

Comment:
Why not just panic inside `try_decode_entire_state` instead of copying this comment everywhere? :see_no_evil: